src/include/kdbopmphm.h

```diff
@@ -22,17 +22,25 @@
  * opmphmRatio should never be less than 1.
```

Comment:
Seems like the C language is quite limited with its static asserts (In C++ with static_assert it works). The check is not really important (its highly unlikely that anyone will ever touch your internal constants), the statement was rather a general recommendation to use formal checks if possible. In general please avoid non-const global variables and defines. Please consider if OPMPHMTUPLE could be `const int` and `opmphmRatio` `const double`?